website/docs/docs/cloud/connect-data-platform/about-connections.md

```diff
@@ -24,6 +24,68 @@ You can connect to your database in dbt Cloud by clicking the gear in the top ri
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connections-legacy-model.png" title="Previous connection model"/>
+
+We are rolling out an important change that moves connection management to the account level. The following connection management section describes these changes. 
+
+This feature is being rolled out in phases over the coming weeks. 
+
+:::
+
+Warehouse connections are an account level resource. As such you can find them under **Accounts Settings** > **Connections**:
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connections-list.png" title="Connection list"/>
+
+Warehouse connections can be re-used across projects. If multiple projects all connect to the same warehouse, you should re-use the same connection in order to streamline your management operations. Connections are assigned to a project via an environment. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connections-new-model.png" title="Connection model"/>
+
+As shown above, a project with 2 environments can target between 1 and 2 different connections. If you want to separate your production environment from your non-production environment, use multiple connections for a single project.
+
+### Migration from project level connections to account level connections
+
+As we roll-out account level connections, you should not expect any interruption of service in your current usage (IDE, CLI, jobs...).
+
+But to fully utilize the value of account level connections, you may have to re-thing how you assign and use connections across projects and environments.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connections-post-rollout.png" title="Typical connection setup post rollout"/>
+
+Please read all of the potential actions below before beginning to work on your connections, as depending on the final outcome you wish to attain you may want to take different first steps.
+
+- Initial clean-up (connection list)
+  - Delete un-used connections (with 0 environment)
+  - Re-name connections to a temporary naming scheme, to better understand where each is used
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connections-post-rollout-2.png" title="Post initial clean-up"/>
+
+- Decide on a granularity 
+  - Define an intent for each connection, usually a combination of warehouse/database instance, intended use (dev/prod…), and administrative surface (which teams/projects will want to change that connection together)
+  - Try to aim for a minimization of the need for local overrides (like extended attributes)
+  - Settle on a naming convention - we recommand you name connections after the server hostname and distinct intent/domain/configuration - it will be easier to re-use connections across projects this way
```

Comment:
```suggestion
  - Come to a consensus on a naming convention.  We recommend you name connections after the server hostname and distinct intent/domain/configuration. It will be easier to reuse connections across projects this way
```